example-apps/react-spa-tanstack-db/src/index.css

```
@@ -0,0 +1,685 @@
+  --ink: #171813;
+  --muted: #69695f;
+  --line: rgba(23, 24, 19, 0.15);
+  --signal: #ff5c35;
+  --signal-dark: #bd3217;
+  --green: #16704e;
+  --yellow: #e8c84a;
+  --panel: rgba(255, 253, 245, 0.86);
+  --serif: "Iowan Old Style", "Palatino Linotype", Palatino, Georgia, serif;
+  --sans: "Avenir Next", Avenir, "Segoe UI", sans-serif;
+  --mono: "SFMono-Regular", Consolas, "Liberation Mono", monospace;
+  font-family: var(--sans);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)